app/src/layouts/common/appSidebar/appSidebar.jsx

```diff
@@ -45,13 +45,25 @@ export const AppSidebar = ({
 }) => {
   const { formatMessage } = useIntl();
   const [isOpenAvatarPopover, setIsOpenAvatarPopover] = useState(false);
+  const [isHoveredUser, setIsHoveredUser] = useState(false);
+
+  const onHoverUser = () => {
+    setIsHoveredUser(true);
+  };
+
+  const onClearUser = () => {
```

Comment:
[ Optional] It seems that this is not a very clear name. Maybe call it something like onUserFocusLeave?